tests/posix/semaphores/src/main.c

```diff
@@ -51,11 +54,10 @@ static void semaphore_test(sem_t *sem)
 
 	zassert_equal(clock_gettime(CLOCK_REALTIME, &abstime), 0, "clock_gettime failed");
 
-	abstime.tv_sec += 5;
+	abstime.tv_sec += WAIT_TIME_MS / MSEC_PER_SEC;
```

Comment:
Is this supposed to be in the second commit?

Reply:
Fixed!